src/customizing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,45 @@
+# Tuning Component Settings
+
+Helm Charts are a unit of Kubernetes manifests that reflect best practices to deploy an application
+or service. Helm is heavily influenced by git, including the
+[formula model](https://github.com/Homebrew/homebrew-core). A Helm chart is to Helm as a Formula
+is to Homebrew.
+
+When you run `helm fetch deis/workflow-beta1`, you can customize the chart with
+`helm edit workflow-beta1`. To customize the respective component, edit
+`manifests/deis-<component>-rc.yaml` and modify the `env` section of the component to tune these
+settings.
+
+For example, to allow only administrators to register new accounts in the controller,
+edit `manifests/deis-controller-rc.yaml` and add the following under the `env` section:
+
+```
+env:
+  - name: REGISTRATION_MODE
+    value: "admin_only"
+```
+
+
+## Customizing the Controller
+
+The following environment variables are tunable for the [Controller][] component:
+
+Setting             | Description
+------------------- | ---------------------------------
+REGISTRATION_MODE   | set registration to “enabled”, “disabled”, or “admin_only” (default: “enabled”)
+GUNICORN_WORKERS    | number of gunicorn workers spawned to process requests (default: 8)
```

Comment:
We only default to `8` if the worker count can't be calculated: https://github.com/deis/controller/blob/master/rootfs/deis/gconf.py